boot/initramfs.go

```diff
@@ -83,3 +86,26 @@ func EnsureNextBootToRunMode(systemLabel string) error {
 	}
 	return bl.SetBootVars(m)
 }
+
+// initramfsReboot triggers a reboot from the initramfs immediately
+var initramfsReboot = func() error {
+	if osutil.IsTestBinary() {
+		panic("initramfsReboot must be mocked in tests")
+	}
+
+	out, err := exec.Command("/bin/reboot").CombinedOutput()
```

Comment:
everything is symlinked I think but I have a slight prefence to use the packaging dir:  /sbin/reboot

Reply:
Sure

Reply:
see other comment though actually